pkg/middleware/default_http_client.go

```diff
@@ -54,9 +54,7 @@ func init() {
 		IdleConnTimeout:       90 * time.Second,
 		TLSHandshakeTimeout:   10 * time.Second,
 		ExpectContinueTimeout: 1 * time.Second,
-		TLSClientConfig: &tls.Config{
-			MinVersion: tls.VersionTLS12,
-		},
+		TLSClientConfig:       &tls.Config{},
```

Comment:
how about just remove this config struct if there is no extra overrides

Reply:
makes sense. Done